### PR TITLE
Updates to v2.9.1 of FullCalendar.

### PR DESCRIPTION
**Pull request by ytrrty**

### CODE REVIEW

File: vendor/assets/javascripts/fullcalendar.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /*!
- * FullCalendar v2.8.0
- * Docs & License: http://fullcalendar.io/
- * (c) 2016 Adam Shaw
+ * <%= meta.title %> v<%= meta.version %>
```
Comment (davidwessman) [inline vendor/assets/javascripts/fullcalendar.js]: Can anyone explain how this works?
(From where is the meta-data defined?)

**Comment by davidwessman** [comment]
Seems very nice 😄 Just wondering about the meta-definitions.

**Comment by gr8bit** [comment]
I'd rather leave all the original FullCalendar files untouched as this gem is only a repackaging.

**Comment by davidwessman** [comment]
@gr8bit this is the original file from Arshaw, I do just not understand how it works!

**Comment by bokmann** [comment]
I do not have the bandwidth to maintain this gem right now, but I don't want to see it die either.  Would any of the commenters on this thread or other users like to take on maintenance and publishing to RubyGems?

**Comment by gr8bit** [comment]
I merged the PR for version 3.0.0 lately - is there still any need for this version (2.9.1) @ytrrty ?

@bokmann I got you covered, but if @davidwessman wants to join the team of maintainers, I'd gladly welcome him! :)

**Comment by davidwessman** [comment]
I would love to join as a maintainer 😄 

And I do think we can close this PR.

**Comment by ytrrty** [comment]
@bokmann Thanks for adding me as a collaborator, happy to help :)
@gr8bit When v3.0.0 was merged v2.9.1 is no longer needed
I'll close this PR.

**Pull request closed without merging**